pmg/static/resources/css/style.scss

```diff
@@ -946,6 +958,34 @@ body.email_alerts {
   max-height: 150px;
 }
 
+.c-m-nav {
```

Comment:
I think this is a bit too short to be meaningful. Perhaps cte-meeting-nav if committee is too long?